src/hotspot/share/memory/metaspace/blockTree.cpp

```diff
@@ -70,9 +84,11 @@ struct BlockTree::walkinfo {
 void BlockTree::verify() const {
   // Traverse the tree and test that all nodes are in the correct order.
 
+  // assert a condition with information about node "n"
+  #define assrt0n(cond, failure_node) assrt0(cond, "Invalid node: " NODE_FORMAT, NODE_FORMAT_ARGS(failure_node))
+
```

Comment:
Do you think this assert is only usable in this method? I would probably be cleaner to move it next to the assertion it chains with.